File: python/w3school/pyMySql/create_table.py
from config.db import pyDb

pyCursor = pyDb.cursor()

sqlCommand = "CREATE TABLE customers(name VARCHAR(255), address VARCHAR(255))"
pyCursor.execute(sqlCommand)

sqlCommand = "CREATE TABLE users(id INT AUTO_INCREMENT PRIMARY KEY, name VARCHAR(255), fav INT)"
pyCursor.execute(sqlCommand)

sqlCommand = "CREATE TABLE products(id INT AUTO_INCREMENT PRIMARY KEY, name VARCHAR(255))"
pyCursor.execute(sqlCommand)

sqlCommand = "CREATE TABLE testing(name VARCHAR(255), address VARCHAR(255))"
pyCursor.execute(sqlCommand)

File: js/nodejs/NodeMySql/CreateTable.js
var config = require('./Settings');
var mysql = require('mysql');

var connection = mysql.createConnection(config.settings);
connection.connect(function(error) {
    if (error) throw error;
    var query = " \
                CREATE TABLE customers \
                (id INT AUTO_INCREMENT PRIMARY KEY, \
                name VARCHAR(255), \
                address VARCHAR(255)) \
            ";
    connection.query(query, function(error, result) {
        if (error) throw error;
        console.log("TABLE CREATED");
    })
})

